common/src/main/java/com/wynntils/features/user/map/MinimapFeature.java

```diff
@@ -258,6 +258,8 @@ private void renderPois(
                     .toList();
             poisToRender.addAll(playerPois);
 
+            poisToRender.addAll(new ArrayList<>(MapModel.getCombatPois()));
```

Comment:
```suggestion
            poisToRender.addAll(apModel.getCombatPois());
```

I'm confident this works